common/app/views/fragments/analytics/google.scala.html

```diff
@@ -103,9 +103,14 @@
         @for(sponsorLogos <- listSponsorLogosOnPage(page)) {
           ga('@{trackerName}.set', 'dimension31', '@Html(sponsorLogos.mkString("|"))');
         }
+
         @for(brandingType <- Commercial.brandingType(page)) {
           ga('@{trackerName}.set', 'dimension42', '@brandingType.name');
         }
+
+        if(document.referrer.includes('oauth') && document.referrer.includes('facebook')) {
```

Comment:
I think we could do more here just to be on the safe side.  Best to check the referrer starts with `https?:\/\/\w*\.?facebook\.com` and contains `dialog/oauth` probably.